### PR TITLE
feat(Sticker): add support for gif stickers

### PR DESCRIPTION
**Pull request by almeidx**
**Please describe the changes this PR makes and why it should be merged:**
Upstream:
- discord/discord-api-docs#5833

Depends on:
- discordjs/discord-api-types#688

**Status and versioning classification:**
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

File: packages/discord.js/src/structures/Sticker.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 'use strict';
 
 const { DiscordSnowflake } = require('@sapphire/snowflake');
-const { Routes, StickerFormatType } = require('discord-api-types/v10');
+const { ImageFormat, Routes, StickerFormatType } = require('discord-api-types/v10');
 const Base = require('./Base');
 const { DiscordjsError, ErrorCodes } = require('../errors');
 
+const stickerFormatExtensionMap = {
```
Comment (kyranet) [inline packages/discord.js/src/structures/Sticker.js:8]:
```suggestion
const StickerFormatExtensionMap = {
```

File: packages/discord.js/src/structures/Sticker.js [hunk 2]
```diff
@@ -164,7 +171,7 @@ class Sticker extends Base {
    * @readonly
    */
   get url() {
-    return this.client.rest.cdn.sticker(this.id, this.format === StickerFormatType.Lottie ? 'json' : 'png');
+    return this.client.rest.cdn.sticker(this.id, stickerFormatExtensionMap[this.format]);
```
Comment (kyranet) [inline packages/discord.js/src/structures/Sticker.js:174]:
```suggestion
    return this.client.rest.cdn.sticker(this.id, StickerFormatExtensionMap[this.format]);
```

**Review by kyranet — COMMENTED** [review]
Minor nits, the map is a pseudo-enum, so it needs to be named in PascalCase, as the rest of the constants from the Constants file (https://github.com/discordjs/discord.js/blob/05a1cbfe5479195b0bc9b6f0971fe39f6af6fd77/packages/discord.js/src/util/Constants.js).

Also, shouldn't we export this (in the `Constants` file) so users can map the extensions themselves?

**Review by kyranet — CHANGES_REQUESTED** [review]
Sorry, I <kbd>Ctrl+Enter</kbd>'d instead of clicking "Request changes". ↑

**Review by kyranet — APPROVED** [review]

**Pull request merged**